### PR TITLE
[CM-1959] Iframe Support for HTML Content 

### PR DESCRIPTION
**Pull request by AbhijeetRanjan308**
## Summary
- Added support for Iframe Contents.

### CODE REVIEW

File: Sources/Views/ALKFriendMessageCell.swift [hunk 1]
```diff
@@ -212,7 +213,19 @@ open class ALKFriendMessageCell: ALKMessageCell {
...
+            iframeView.topAnchor.constraint(
+                equalTo: bubbleView.topAnchor
+            ),
+            iframeView.bottomAnchor.constraint(
+                equalTo: bubbleView.bottomAnchor
+            ),
+            iframeView.trailingAnchor.constraint(
+                equalTo: bubbleView.trailingAnchor
+            ),
+            iframeView.leadingAnchor.constraint(
```
Comment (Sathyan-KM) [inline Sources/Views/ALKFriendMessageCell.swift:225]: make it in single line

File: Sources/Views/ALKMessageBaseCell.swift [hunk 2]
```diff
@@ -168,7 +178,8 @@ open class ALKMessageCell: ALKChatBaseCell<ALKMessageViewModel> {
         mentionStyle: Style
     ) {
         self.viewModel = viewModel
-
+        iframeView.isHidden = true
+        iframeWidth.isActive = false
```
Comment (Sathyan-KM) [inline Sources/Views/ALKMessageBaseCell.swift:182]: can't we move it inside the initialisation of iframeview similar to is opaque

**Review by Sathyan-KM — CHANGES_REQUESTED** [review]
Check the scrolling test case as well

File: Sources/Views/ALKMessageBaseCell.swift (hunk 2 above)
Comment (AbhijeetRanjan308) [reply Sources/Views/ALKMessageBaseCell.swift:182]: its need to be there as iframeView is initialised only one time and when ever there is some iframe msg comes that iframeView will be set to not hidden state that can effect when there is non iframe msg comes up.
For iFrameWidth that is the constraint that also needed to be there as update will handle that it's active or not.

**Review by Sathyan-KM — APPROVED** [review]

**Pull request merged**